.github/SPRINT_TEMPLATES/sprint-goal.md

```diff
@@ -0,0 +1,70 @@
+name: Sprint goal
+about: Open a new Sprint Goal
+title: ''
+labels: ''
+assignees: ''
+---
+## Description
+
+Write a description of the issue
+
+
+### Goal and Acceptance Criteria
+Release X.Y.Z-rc
+
+[ReleaseVersion] - [DATE] - [Sprint Goal]
+- [DeliveryType][DeliveryDomain] ..delivery description (actor -> action)
```

Comment:
Shouldn't this just be a list of stories in the sprint?

Reply:
prefer elevated acceptance criteria per release